packages/teleterm/src/ui/ClusterLogout/useClusterLogout.ts

```diff
@@ -18,6 +18,7 @@ export function useClusterLogout({ clusterUri, onClose, clusterTitle }: Props) {
         await ctx.workspacesService.setActiveWorkspace(null);
       }
     }
+    ctx.workspacesService.removeWorkspace(clusterUri);
```

Comment:
Removing a workspace has nothing to do with these bugs, but we should make the clean-up

Reply:
Does this take care of gravitational/webapps.e#154? I suppose only in part, because it'll remove the docs but probably won't take care of the other stuff (I don't actually know if there's any other stuff to take care of, perhaps any open connections?).

Reply:
This PR had to resolve this issue: `Cluster with URI /clusters/teleport-local does not exist`, but I didn't commit it :/ (fix is now in `workspacesService` line 141).
Unfortunately, the line you mentioned won't resolve any issues from the linked issue. The problem is in `StatePersitenceService` and its messy logic (it doesn't remove the workspace from the `app_state.json`)